### PR TITLE
fix: Travis Jest test not downloading mongo binary fast enough

### PR DESCRIPTION
**Pull request by karrui**
## Problem

Our Jest tests are failing and require multiple retries due to tests encountering timeout due to only starting the mongo binary download in the tests themselves (if the binary does not exist). 

This change allows for the binary to be downloaded if not yet cached before the individual tests even run

Also remove travis_retry, since the flaky form model test has been fixed.

See:
Screenshot 2020-12-09 at 11 09 29 AM

### CODE REVIEW

File: tests/database.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ class MemoryDatabaseServer {
   constructor() {
     this.mongod = new MongoMemoryServer({
       binary: {
-        version: process.env.MONGO_BINARY_VERSION || '3.6.12',
+        version: process.env.MONGO_BINARY_VERSION || '4.4.1',
```
Comment (karrui) [inline tests/database.js:7]: This shouldn't matter btw, since our .test-full-env is still promoting 3.6.1x

**Review by mantariksh — COMMENTED** [review]
sorry, I don't really understand what's going on here. `jest-db` only exists to export `getConnectionString` which uses `getUri` under the hood, and `CustomEnvironment` only exists to set the `__DB_URL__` global. Instead of doing all this, why not just initialise `MongoMemoryServer` in `jest-db.ts` and get the connection string using `getUri`?

**Comment by karrui** [comment]
> sorry, I don't really understand what's going on here. `jest-db` only exists to export `getConnectionString` which uses `getUri` under the hood, and `CustomEnvironment` only exists to set the `__DB_URL__` global. Instead of doing all this, why not just initialise `MongoMemoryServer` in `jest-db.ts` and get the connection string using `getUri`?

the problem with this is that the downloading of the binary (if the binary is unavailable) will only start when tests are being run, and the first jest test will fail if this download takes more than 5 seconds, as seen in the screenshot.

This PR is actually the canonical way to instantiate a database, see https://github.com/shelfio/jest-mongodb/blob/master/environment.js

**Comment by karrui** [comment]
I'll put it in a setup file for jest instead of the environment!

**Review by mantariksh — APPROVED** [review]

**Pull request merged**